Lib/test/test_time.py

```diff
@@ -159,6 +159,18 @@ def test_sleep(self):
         self.assertRaises(ValueError, time.sleep, -1)
         time.sleep(1.2)
 
+    def test_sleep_until(self):
+        start = time.time()
+        deadline = start + 2
+        time.sleep_until(deadline)
+        stop = time.time()
+        delta = stop - deadline
+        # cargo-cult these 50ms from test_monotonic (bpo-20101)
+        self.assertGreater(delta, -0.050)
+        # allow sleep_until to take up to 1s longer than planned
+        # (e.g. in case the system is under heavy load during testing)
+        self.assertLess(delta, 1.000)
```

Comment:
In my experience, such test tends to fail on busy buildbots. I would prefer to not test the "performance" (accuracy) of the function in such unit test.

I'm not sure that it's useful to write an unit test for this function. It's very hard to write a reliable on the clock accuracy.

What if the system clock changes during the test? :-(

Reply:
Yes, I agree, and I actually already had to increase that final `assertLess` because the test failed on a busy runner with a smaller value. When writing the test I took my cue from tests like [`test_monotonic`](https://github.com/python/cpython/blob/96110f85/Lib/test/test_time.py#L478), but I'm also fine with simplifying the test case, let me know.